src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.cs

```diff
@@ -59,6 +58,322 @@ public object ScrollOwner
 
 		private ScrollViewer Scroller => ScrollOwner as ScrollViewer;
 
+		internal object ManagedContent
```

Comment:
We already have a `RealContent` property in `ScrollContentPresenter.Native.cs` could we please merge them? 

I would really try to keep the "native" support in the `ScrollContentPresenter.Native.cs` file (so we never have to test the `NativeScrollContentPresenter` - BTW we should use the interface), so my recommendation would be to add a property `RealContent => Content` in the `ScrollContentPresenter.Managed.cs` and  `ScrollContentPresenter.wasm.cs` files.

About the naming even if I don't like the `RealContent`, for me `ManagedContent` means that this "content" is a "managed" element, which is not necessarily the case (especially when dealing with `ListView`).

Also could we keep that property `private` ?